src/utils/ClientUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseAbstractClient } from "../clients/BaseAbstractClient";
+import assert from "assert";
+
+/**
+ * Asserts that the clients are updated
+ * @param clients The clients to check
+ * @throws AssertionError if the clients are not updated
+ */
+export function assertClientsAreUpdated(...clients: (BaseAbstractClient | undefined | null)[]): void {
+  clients.forEach((client) => !!client && assert(client.isUpdated, `${client.clientName ?? "Client"} is not updated`));
```

Comment:
What does double ! do?

Reply:
`isDefined` is preferred: https://github.com/across-protocol/sdk-v2/pull/359/commits/8445cedff96014fb5c510fe7f6926d7be5cd06c4